hybridse/src/planv2/ast_node_converter.cc

```diff
@@ -2133,6 +2148,81 @@ base::Status ConvertAlterUserStatement(const zetasql::ASTAlterUserStatement* roo
     return base::Status::OK();
 }
 
+base::Status ConvertGrantStatement(const zetasql::ASTGrantStatement* root, node::NodeManager* node_manager,
+                                   node::GrantNode** output) {
+    CHECK_TRUE(root != nullptr, common::kSqlAstError, "not an ASTGrantStatement");
+    std::vector<std::string> target_path;
+    CHECK_STATUS(AstPathExpressionToStringList(root->target_path(), target_path));
+    std::optional<std::string> target_type = std::nullopt;
+    if (root->target_type() != nullptr) {
+        target_type = root->target_type()->GetAsString();
+    }
+
+    std::vector<std::string> privileges;
+    std::vector<std::string> grantees;
+    for (auto privilege : root->privileges()->privileges()) {
+        if (privilege == nullptr) {
+            continue;
+        }
+
+        auto privilege_action = privilege->privilege_action();
+        if (privilege_action != nullptr) {
+            privileges.push_back(privilege_action->GetAsString());
+        }
+    }
+
+    for (auto grantee : root->grantee_list()->grantee_list()) {
+        if (grantee == nullptr) {
+            continue;
+        }
+
+        std::string grantee_str;
+        CHECK_STATUS(AstStringLiteralToString(grantee, &grantee_str));
+        grantees.push_back(grantee_str);
+    }
+    *output = node_manager->MakeNode<node::GrantNode>(target_type, target_path.at(0), target_path.at(1), privileges,
```

Comment:
what if `target_path` size = 1 ?

Reply:
in grant statements the target_path is required to have 2 elements: database and target